var/spack/repos/builtin/packages/portage/package.py

```diff
@@ -51,6 +51,10 @@ def cmake_args(self):
         else:
             options.append('-DENABLE_MPI=OFF')
 
+        options.append('-DBLA_VENDOR={0}'.format(self.spec['blas'].name.upper()))
```

Comment:
Typo? BLA -> BLAS?

Reply:
No, it is really `BLA_VENDOR`.

Reply:
The example code for https://cmake.org/cmake/help/v3.14/module/FindLAPACK.html is:
```cmake
set(BLA_VENDOR Intel10_64lp)
find_package(LAPACK)
```